hoot-services/src/main/java/hoot/services/utils/DbUtils.java

```diff
@@ -215,8 +215,12 @@ public static void deleteTables(List<String> tables) throws SQLException {
                 String sql = "DROP TABLE IF EXISTS \"" + table + "\"";
                 try (PreparedStatement stmt = conn.prepareStatement(sql)) {
                     stmt.execute();
+                    stmt.close();
+                    
                 }
+                conn.commit();    
```

Comment:
Should we commit inside the try?
Or move the commit outside the for loop?
Either one of those makes a bit more sense than where it is now.